dev/package-examples/endpoint-0.0.1/dataset/endpoint.metadata/fields/fields.yml

```diff
@@ -0,0 +1,200 @@
+- name: "@timestamp"
```

Comment:
Same comment as above for the path.

Reply:
I remember also a discussion around endpoint only using one index? But having 2 datasets would mean 2 indices?

Reply:
Gotcha I'll change the dataset name.

So there will be one index for events and alerts (events) and then another index for the metadata of an endpoint for now.

Reply:
Got it, then this makes sense. 

This raises the question how the metadata index will be called. `events-endpoint.metadata-*`? If yes, the name here is correct.

Reply:
So this one will actually be `metrics-endpoint.metadata-*` (the type is `metrics`, this is information about endpoints and their status etc) Do you think `endpoint.metadata` is reasonable name? I figure `metrics-metadata` wasn't super clear on what it was referring to, that's why I add the `endpoint` in there.

If we still with `endpoint.metadata` would it be better to be consistent in the dataset directory and leave the events one as `endpoint.events`?

Reply:
Agree that we can't really use `metrics-metadata` as this is too generic and not clear it would belong to endpoint. 

By default we will have that the dataset id = {package.name}.{directory.name}. So in the nginx case we have access, error and stubstatus directories. The problem above is that you only have `events-endpoint` which means the above does not work as the package is already endpoint. You will have to set id in any case. I would probably call the directory è
`events`, and the other one `metadata`.

Reply:
Ah ok I see. I'll rename the directories and set the `id` field. Should I rebase off of https://github.com/elastic/package-registry/pull/176 ?

Reply:
I suggest lets get this in as is and then adjust it again as soon as #176 lands.

Reply:
Ok sounds good, I moved the datasets to `events` and `metadata`. I'm good to merge. @james-elastic could I get a 👍